src/exec/src/bin/occlum_exec_client.rs

```diff
@@ -269,14 +271,21 @@ fn main() -> Result<(), i32> {
         .map(|(key, val)| format!("{}={}", key, val))
         .collect();
 
-    let mut sock_file = String::from(args[0].as_str());
-    let sock_file = str::replace(
-        sock_file.as_mut_str(),
+    let mut sockfile = String::from(args[0].as_str());
+    let sockfile = str::replace(
+        sockfile.as_mut_str(),
         DEFAULT_CLIENT_FILE,
         DEFAULT_SOCK_FILE,
     );
 
-    let client = OcclumExecClient::new_plain_unix(&sock_file, ClientConf::new())
+    let mut hasher = DefaultHasher::new();
+    sockfile.hash(&mut hasher);
+
+    let mut sockfile = String::from("/tmp/occlum_");
```

Comment:
`/tmp` may not be a good place for UDS files since OS/admins may remove any file within it at any moment. Can we use e.g. `/run` instead?
See FHS suggestions [/run](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s15.html) and [/tmp](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s18.html) for further details.

Reply:
Thanks. Change the path.